tests/test_config_profiles.py

```
@@ -0,0 +1,450 @@
+        """All 8 profiles should apply successfully."""
+        for name in PROFILES:
+            success, errors = apply_profile(name, config_path=temp_config.config_path)
+            assert success, f"Profile '{name}' failed: {errors}"
+
+
+# ---------------------------------------------------------------------------
+# Profile create test
+# ---------------------------------------------------------------------------
+
+class TestProfileCreate:
+    def test_create_from_current_config(self, temp_config, monkeypatch):
```

Ruff (0.15.20)

[warning] Unused method argument: `temp_config`

(ARG002)